README.md

```diff
@@ -15,16 +15,16 @@ Miniforge installers are available here: https://github.com/conda-forge/miniforg
 
 #### Miniforge3
 
-Latest installers with Python 3.10 `(*)` in the base environment:
+Latest installers with Python 3.12 `(*)` in the base environment:
 
-| OS      | Architecture          | Download  |
-| --------|-----------------------|-----------|
-| Linux   | x86_64 (amd64)        | [Miniforge3-Linux-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh) |
-| Linux   | aarch64 (arm64) `(**)` | [Miniforge3-Linux-aarch64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh) |
-| Linux   | ppc64le (POWER8/9)    | [Miniforge3-Linux-ppc64le](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-ppc64le.sh) |
-| OS X    | x86_64                | [Miniforge3-MacOSX-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh) |
-| OS X    | arm64 (Apple Silicon) `(***)` | [Miniforge3-MacOSX-arm64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh) |
-| Windows | x86_64                | [Miniforge3-Windows-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe) |
+| OS      | Architecture                  | Minimum Version  | Download  |
+| --------|-------------------------------|------------------|-----------|
+| Linux   | x86_64 (amd64)                | glibc >= 2.17    | [Miniforge3-Linux-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh) |
+| Linux   | aarch64 (arm64) `(**)`        | glibc >= 2.17    | [Miniforge3-Linux-aarch64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh) |
+| Linux   | ppc64le (POWER8/9)            | glibc >= 2.17    | [Miniforge3-Linux-ppc64le](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-ppc64le.sh) |
+| OS X    | x86_64                        | macOS >= 10.13   | [Miniforge3-MacOSX-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh) |
+| OS X    | arm64 (Apple Silicon) `(***)` | macOS >= 11.0    | [Miniforge3-MacOSX-arm64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh) |
```

Comment:
```suggestion
| OS X    | arm64 (Apple Silicon)         | macOS >= 11.0    | [Miniforge3-MacOSX-arm64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh) |
```

Reply:
Do we still need that footnote?

> `(***)` Apple silicon builds are experimental and haven't had testing like the other platforms.

Reply:
We don't have CI for macOS ARM

That said, maybe we can relax this note a bit

Reply:
I'm not opposed to this, but I think a new PR is warranted if we want to review the language around OSX + arm.

Lets start the builds for this as they take a while already.